mvpa2/testing/tools.py

```diff
@@ -61,23 +62,28 @@ class SkipTest(Exception):
     assert_string_equal)
 
 
+
```

Comment:
Hmm, this seems a limitation of PyCharm's autoformatting. There is one setting that applies to both top-level functions and classes at the same time. The code guidelines say to use 3 empty lines around classes but 2 around functions. Should we consider to have 2 lines also for classes?